app/controllers/to_change_everything_controller.rb

```diff
@@ -0,0 +1,17 @@
+class ToChangeEverythingController < ApplicationController
+  layout 'to_change_everything', only: [:show]
+  skip_before_action :set_pinned_pages, only: [:show]
+
+  def show
+    @locale = params[:lang]
+
+    if I18n.available_locales.map(&:to_s).include?(@locale)
```

Comment:
`I18n.available_locales` returns more locales than there are TCE yaml translations for.

```ruby
I18n.available_locales.map(&:to_s)
#=> ["en", "da", "de", "fr", "nb", "nl", "no", "pl", "pt-BR", "ru", "sv", "es", "si", "espanol"]
```

Maybe instead of using that method, I can hardcode in:

```ruby
TO_CHANGE_ANYTHING_YAMLS = ["espanol"].freeze
```

and then do
```ruby
 if TO_CHANGE_ANYTHING_YAMLS.include?(@locale)
...
...
```

Reply:
@astronaut-wannabe  Where is the list of langs coming from in this?

```ruby
I18n.available_locales.map(&:to_s)
```

Reply:
@veganstraightedge it comes from `I18n.load_path`, which (at least on my local machine) includes some gem paths

You _can_ whitelist locales in the config like this:

```
# Whitelist locales available for the application
I18n.available_locales = [:en, :es, :espanol]
```

but that won't really solve the issue (which is that right now going to something like https://crimethinc-staging-pr-577.herokuapp.com/tce/en )

Reply:
Gotcha. In that case, I think explicitly listing the TCE translations is a good approach.

`TO_CHANGE_ANYTHING_YAMLS = ["espanol"].freeze`